monkestation/code/game/objects/items/ancient_artifacts.dm

```diff
@@ -0,0 +1,455 @@
+//Daemon Mask
+/obj/item/clothing/mask/daemon_mask
```

Comment:
Overall opinion of this one is that it's far too powerful for an easy cargo purchase.
I'd make it a Curator restricted TC item.

Reply:
Do you care if it's in this PR or should it be a separate one